core/scss/components/drawer/drawer.scss

```diff
@@ -0,0 +1,78 @@
+ /** @component drawer */
+@include exports('md-drawer') {
+
+  .#{$prefix}-drawer {
+    z-index: $drawer-zindex;
+    position: fixed;
+    overflow: auto;
+    transition-duration: 0.6s;
+    background-color: $drawer-background-color;
+    border: 1px solid $drawer-border-color;
+    box-shadow: 0px 1px 2px 1px rgba(52, 53, 55, 0.18);
+    @include vr-spacing(p, 1);
+    &.#{$prefix}-drawer--right {
```

Comment:
Should these all be nested @pauljeter?  I would only nest when certain properties can't be overwritten otherwise to make it easier for adopters.

Reply:
Yes. Ben and I have discussed de-nesting as many classes as possible.